tests/Builder/PackagesBuilderDumpTest.php

```diff
@@ -83,7 +83,8 @@ public function testNominalCase($providers = false)
                 $packageName = key($arrayPackages);
                 $arrayPackages[$packageName][$i . '.0']['uid'] = 1;
                 $hash = current($packagesJson['providers'][$packageName]);
-                $includeJson = str_replace(['%package%', '%hash%'], [$packageName, $hash], $packagesJson['providers-url']);
+                $includeJson = str_replace(['%package%', '%hash%'], [$packageName, $hash],
+                    $packagesJson['providers-url']);
```

Comment:
unrelated change?

Reply:
Yeah, I think your editor did something quirky here. Can you undo this?

Reply:
undone